docs/site/pages/tools/devtools.mdx

```diff
@@ -0,0 +1,67 @@
+# Devtools
+
+## Introduction
+https://github.com/player-ui/browser-devtools 
+Our Devtools player extension has been redesigned to work with manifest v3. It leverages plugins to handle communicated between the inspected Player-UI instance and the devtools extension panel. 
```

Comment:
Do we even want to call out that it was "redesigned"? Given that the current version was never published is it even worth mentioning?